### PR TITLE
Add handmade monitors to datadog-monitors

### PR DESCRIPTION
**Pull request by rattboi**
Now that we have a facility to deploy our monitors, we should move all
our hand-rolled monitors into here to be versioned.

These monitors will be deployed during the bastion cronjob.

Signed-off-by: Bradon Kanyid <rattboi@gmail.com>

### CODE REVIEW

File: monitors.yml [hunk 1]
```diff
@@ -1,24 +1,142 @@
...
+    multi: true
+    options:
+      notify_no_data: false
+      thresholds:
+        critical: 1
+        ok: 1
+        warning: 1
+    query: '"process.up".over("host:zuul","process:mysql").by("host","process").last(2).count_by_status()'
+    type: service check
+  - message: "Gearman went down!\n\n{{#is_alert}}\nLog onto host and fix it!\n{{/is_alert}}\
```
Comment (jamielennox) [inline monitors.yml]: Use 
```
message: | 
  Gearman went down!

  {{ #is_alert }}
  Log onto host
 ```

style syntax here. It'll be easier

File: monitors.yml [hunk 2]
```diff
@@ -1,24 +1,142 @@
...
   monitors:
-    multi: true
     options:
       locked: true
+      new_host_delay: 300
+      no_data_timeframe: 2
+      notify_audit: false
+      renotify_interval: 0
+      silenced: {}
+      timeout_h: 0
```
Comment (jamielennox) [inline monitors.yml:10]: Do all of these make sense to be in the global options like this? 

I was thinking that maybe we could do additional options by tag or something later, but for now being unset should be the default for most of these right?

**Review by jamielennox — CHANGES_REQUESTED** [review]

File: monitors.yml (hunk 1 above)
Comment (rattboi) [reply monitors.yml]: I just used the generated output. I can fix this.

File: monitors.yml (hunk 2 above)
Comment (rattboi) [reply monitors.yml:10]: These were all the ones that were common in the generated output. These all exist from 'Datadog-builder init'

**Review by jamielennox — APPROVED** [review]

**Pull request merged**